applications/luci-app-attendedsysupgrade/htdocs/luci-static/resources/view/attendedsysupgrade/overview.js

```diff
@@ -608,7 +608,7 @@ return view.extend({
 			E(
 				'p',
 				_(
-					'The attended sysupgrade service allows to easily upgrade vanilla and custom firmware images.'
+					'The attended sysupgrade service allows to upgrade easily vanilla and custom firmware images.'
```

Comment:
IMHO, the old version is easier to comprehend.

Reply:
https://grammarist.com/grammar/split-infinitives/ 
Acceptable.

Reply:
( Although discouraged. Since it is clear what the word easily (adverb) modifies. )

Reply:
It's a moot point now that the changes have been merged, but your link would imply the better version would be: `The attended sysupgrade service allows to upgrade vanilla and custom firmware images easily.` not `The attended sysupgrade service allows to upgrade easily vanilla and custom firmware images.`.